Debug build fails.

Attempting to build a debug-enabled version of spglib fails with

../libsymspg.so.2.6.1: error: undefined reference to 'warning_memory'

See attached log.
[spglib-debug.log](https://github.com/user-attachments/files/19866513/spglib-debug.log)

--- a/src/debug.c
+++ b/src/debug.c
@@ -88,6 +88,9 @@ void warning_print(char const* format, ...) {
     vfprintf(stderr, format, argptr);
     va_end(argptr);
 }
+void warning_memory(char const* what) {
+    warning_print("Spglib: Memory could not be allocated: %s\n", what);
+}
 #endif
 
 #ifdef SPGINFO

--- a/src/debug.h
+++ b/src/debug.h
@@ -20,6 +20,7 @@ static inline void debug_print(char const* format, ...) {}
 #endif
 #ifndef SPGWARNING
 static inline void warning_print(char const* format, ...) {}
+static inline void warning_memory(char const* what) {}
 #endif
 #ifndef SPGINFO
 static inline void info_print(char const* format, ...) {}
@@ -38,6 +39,4 @@ void debug_print_vectors_with_label(double const a[][3], int const b[],
                                     int size);
 
 // Common messages
-inline void warning_memory(char const* what) {
-    warning_print("Spglib: Memory could not be allocated: %s\n", what);
-}
+void warning_memory(char const* what);

